elevate-cpanel

```diff
@@ -6457,7 +6457,8 @@ EOS
             qr/^cloudlinux-ea4-rollout(?:-[0-9]+)?$/,
             'cl-ea4',
             qr/^cl-mysql(?:-meta)?/,
-            'mysqclient', 'mysql-debuginfo'
+            'mysqclient', 'mysql-debuginfo',
```

Comment:
Please edit `lib/Elevate/OS/CloudLinux7.pm` with your change and then run `make`. The file you edited is generated by the makefile based off the contents of `lib/`

Thanks!